src/infi/clickhouse_orm/query.py

```diff
@@ -392,6 +393,9 @@ def as_sql(self):
             if self._grouping_with_totals:
                 sql += ' WITH TOTALS'
 
+        if self._having:
+            sql += '\nhaving ' + self.having_as_sql()
```

Comment:
it makes sense to use caps i.e. `HAVING` in order to keep the style

Reply:
ok,, I change it.